### PR TITLE
Modifed Parse to handle Blank Nodes.

### PR DESCRIPTION
**Pull request by palimarrao**
I did not do too much of error checking in the Parse method since the lexer should check for valid blank nodes.

### CODE REVIEW

File: triple/node/node.go [hunk 1]
```diff
@@ -74,25 +79,34 @@ func (n *Node) String() string {
...
+		if raw[len(raw)-1] != '>' {
+			return nil, fmt.Errorf("node.Parser: pretty printing should finish with '>' in %q", raw)
+		}
+		id, err := NewID(raw[idx+1 : len(raw)-1])
+		if err != nil {
+			return nil, fmt.Errorf("node.Parser: invalid ID in %q, %v", raw, err)
+		}
+		return NewNode(t, id), nil
+	case underscore:
+		id := ID(raw[2:len(raw)])
```
Comment (xllora) [inline triple/node/node.go]: id := NewID(raw[2:len(raw)]0

File: triple/node/node.go [hunk 2]
```diff
@@ -74,25 +79,34 @@ func (n *Node) String() string {
...
+		t :=  Type("/_")
+		return NewNode(&t, &id), nil
```
Comment (xllora) [inline triple/node/node.go]: return NewNode(&t, id), nil

Both changes for consistency with the code below. I would also suggest use NewType.

File: triple/node/node.go (hunk 1 above)
Comment (palimarrao) [inline triple/node/node.go]: Done.

File: triple/node/node.go (hunk 2 above)
Comment (palimarrao) [inline triple/node/node.go]: Done.

**Comment by palimarrao** [comment]
I have addressed the comments. PTAL, thanks!

**Pull request merged**